lib/maths/CBoostedTreeImpl.cc

```diff
@@ -1512,6 +1514,7 @@ const std::string DOWNSAMPLE_FACTOR_TAG{"downsample_factor"};
 const std::string ENCODER_TAG{"encoder"};
 const std::string ETA_GROWTH_RATE_PER_TREE_TAG{"eta_growth_rate_per_tree"};
 const std::string ETA_OVERRIDE_TAG{"eta_override"};
+const std::string ETA_GROWTH_PER_TREE_OVERRIDE_TAG{"eta_growth_per_tree"};
```

Comment:
is `rate` missing intentionally?

Reply:
No. Agreed it makes sense to be consistent.

Reply:
Fixed in [e9ba4f5](https://github.com/elastic/ml-cpp/pull/1683/commits/e9ba4f50624d2e832ce41e0e8ec459c2855bbaf8).